azure-pipelines.yml

```diff
@@ -0,0 +1,63 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+- master
+
+
+jobs:
+- job: Test
+  strategy:
+    matrix:
+      node_12: 
+        node_version: '12'
+        TEST_REPOSITORY: 'CORE'
+      node_10:
+        node_version: '10'
+        TEST_REPOSITORY: 'CORE'
+      node_8:
+        node_version: '8'
+        TEST_REPOSITORY: 'CORE'
+  variables:
+    TEST_REPOSITORY: $(test_repository)
+  pool:
+    vmImage: 'ubuntu-latest'
+  steps:
+    - task: NodeTool@0
+      inputs:
+        versionSpec: $(node_version)
+      displayName: 'Install Node.js'
+    - script: |
+        yarn
+      displayName: 'Install dependencies'
+    - script: |
+        yarn run ci
+      displayName: 'Run CI'
+
+- job: Extended_tests
+  strategy:
+    matrix:
+      jhipster-1:
+        TEST_REPOSITORY: 'JHIPSTER-1'
```

Comment:
can we reference this variable in the script command below.
So instead of using an env variable we would pass the name of the yarn script to run?

So we would execute:
- lerna run ${SCRIPT_NAME}

and the value would be "e2e-A" or "e2e-B" for example.

I prefer to avoid depending env variables because it makes it harder to execute and reproduce the build locally.

Reply:
I have made the modifications to not use env variables.